dingo/bindings/hmc_sampling.h

```diff
@@ -0,0 +1,82 @@
+#include "ode_solvers/ode_solvers.hpp"
+#include "random_walks.hpp"
+#include <list>  
+using namespace std;
```

Comment:
Since this is included in bindings.h, plz don't use std as a namespace and add `std::` where it is necessary.